payroll-processor-functions/src/payroll-processor-functions/payroll-processor-functions.csproj

```diff
@@ -21,7 +21,7 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
     <None Update="local.settings.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Was this causing issues?

Reply:
nope i didn't mean to check this in, it was a test on the app settings